package/MDAnalysis/core/AtomGroup.py

```diff
@@ -4497,7 +4497,7 @@ def load_new(self, filename, **kwargs):
 
         if not reader:
             # Check if we need to use Chain reader
-            if util.iterable(filename):
+            if not isinstance(filename, util.NamedStream) and util.iterable(filename):
```

Comment:
The way it was before is that `util.iterable` would return False if applied to a `NamedStream` because it is an instance of `basetring`. So `util.iterable` could continue to return False when applied to `NamedStream`. It has the advantage of not changing the current behavior.

Yet, if you want to clarify what it means to iterate on a `NamedStream`, you are very welcome.